dev-libs/efl/efl-1.20.7-r1.ebuild

```diff
@@ -0,0 +1,320 @@
+IUSE="avahi +bmp dds connman debug doc drm +eet egl examples fbcon +fontconfig fribidi gif gles glib gnutls gstreamer harfbuzz hyphen +ico ibus ivi jpeg2k libressl libuv luajit neon nls opengl ssl pdf physics pixman postscript +ppm +psd pulseaudio raw scim sdl sound static-libs svg systemd tga tiff tslib unwind v4l valgrind vlc vnc wayland webp X xcf xim xine xpresent xpm"
+
+REQUIRED_USE="
+	fbcon? ( !tslib )
+	gles? (
+		|| ( X wayland )
+		!sdl
+		egl
+	)
+	gnutls? ( ssl )
+	ibus? ( glib )
+	libressl? ( ssl )
+	opengl? (
+		|| ( X sdl wayland )
+		!egl
+	)
+	pulseaudio? ( sound )
+	sdl? ( opengl )
+	vnc? ( X fbcon )
+	wayland? ( egl gles !opengl )
+	xim? ( X )
+"
+
+RDEPEND="
+	avahi? ( net-dns/avahi )
+	connman? ( net-misc/connman )
+	drm? (
+		dev-libs/libinput
+		media-libs/mesa[gbm]
+		x11-libs/libdrm
+		x11-libs/libxkbcommon
+	)
+	egl? ( media-libs/mesa[egl] )
+	fontconfig? ( media-libs/fontconfig )
+	fribidi? ( dev-libs/fribidi )
+	gif? ( media-libs/giflib:= )
+	gles? ( media-libs/mesa[gles2] )
+	glib? ( dev-libs/glib:2 )
+	gstreamer? (
+		media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0
+	)
+	harfbuzz? ( media-libs/harfbuzz )
+	hyphen? ( dev-libs/hyphen )
+	ibus? ( app-i18n/ibus )
+	jpeg2k? ( media-libs/openjpeg:0 )
+	libuv? ( dev-libs/libuv )
+	luajit? ( dev-lang/luajit )
+	!luajit? ( dev-lang/lua:* )
+	nls? ( sys-devel/gettext )
+	pdf? ( app-text/poppler[cxx] )
+	physics? ( sci-physics/bullet )
+	pixman? ( x11-libs/pixman )
+	postscript? ( app-text/libspectre )
+	media-libs/libpng:0=
+	pulseaudio? ( media-sound/pulseaudio )
+	raw? ( media-libs/libraw )
+	scim? ( app-i18n/scim )
+	sdl? (
+		media-libs/libsdl2
+		virtual/opengl
+	)
+	sound? ( media-libs/libsndfile )
+	ssl? (
+		gnutls? ( net-libs/gnutls )
+		!gnutls? (
+			!libressl? ( dev-libs/openssl:0= )
+			libressl? ( dev-libs/libressl:0= )
+		)
+	)
+	svg? (
+		gnome-base/librsvg
+		x11-libs/cairo
+	)
+	systemd? ( sys-apps/systemd )
+	tiff? ( media-libs/tiff:0= )
+	tslib? ( x11-libs/tslib )
+	unwind? ( sys-libs/libunwind )
+	valgrind? ( dev-util/valgrind )
+	vlc? ( >=media-video/vlc-3.0.0 )
+	vnc? ( net-libs/libvncserver )
+	wayland? (
+		dev-libs/wayland
+		media-libs/mesa[gles2,wayland]
+		x11-libs/libxkbcommon
+	)
+	webp? ( media-libs/libwebp )
+	X? (
+		media-libs/freetype
+		x11-libs/libX11
+		x11-libs/libXcomposite
+		x11-libs/libXcursor
+		x11-libs/libXdamage
+		x11-libs/libXext
+		x11-libs/libXfixes
+		x11-libs/libXinerama
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXScrnSaver
+		x11-libs/libXtst
+
+		opengl? (
+			x11-libs/libX11
+			x11-libs/libXrender
+			virtual/opengl
+		)
+
+		gles? (
+			x11-libs/libX11
+			x11-libs/libXrender
+			virtual/opengl
+			xpresent? ( x11-libs/libXpresent )
+		)
+	)
+
+	xine? ( media-libs/xine-lib )
+	xpm? ( x11-libs/libXpm )
+
+	sys-apps/dbus
+	sys-apps/util-linux
+	sys-libs/zlib
+	virtual/jpeg:0=
+
+	!media-libs/elementary
```

Comment:
blockers should not end up in `DEPEND`. Please make this var be `COMMON_DEPEND` and split out the blockers in a separate `RDEPEND` definition below.